data/human/culture conversations.txt

```diff
@@ -344,11 +344,21 @@ mission "Forest Fire"
 mission "Local Politics"
 	minor
 	source
-		government "Free Worlds"
+		government "Pirate" "Republic"
 		attributes "frontier"
 		not attributes "station"
 	to offer
 		random < 1
+		or
+			and
+				not "flagship planet: Albatross"
+				not "flagship planet: Bloodsea"
+			and
+				not "event: albatross joins free worlds"
+				has "flagship planet: Albatross"
+			and
+				not "event: bloodsea joins free worlds"
+				has "flagship planet: Bloodsea"
```

Comment:
These events change change the governments to Free Worlds, so by not including that government in the source filter, you've already prevented the mission from offering on these planets after these events occur.